libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -145,6 +145,7 @@ define HAL_PARACHUTE_ENABLED 0
 
 # save FLASH, but leave above when flash issue is fixed
 include ../include/minimize_fpv_osd.inc
+include ../include/no_bootloader_DFU.inc
```

Comment:
Just to be clear, it won't build without this?

Reply:
it probably would, but eventually won't..not being able to update the bootloader from firmware when the board had a boot button has been discussed in previous devcalls and the consensus was that it was not needed on those boards since a means to update it, when and if a necessary bootloader change occurs, is included on the board...unlike boards like CubeOrange